cli/src/main/package/ide

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Check if arguments are given
+if [ -n "${1}" ]; then
+  # Call native ideasy.exe with user-provided arguments
+  ./ideasy "$@"
```

Comment:
added error handling here:
```
rc=$?
if [ "${rc}" != 0 ]; then
  exit ${rc}
fi
```
As our script will be sourced, we might want to avoid the local variable. I have not tested if $? changes by `if` statement... But nice to have for further improvements after merge...

Reply:
I changed it using `set -e` to solve the problem.